R/getPrevalence.R

```diff
@@ -228,7 +228,7 @@ setMethod("getPrevalence", signature = c(x = "ANY"), function(
 .agg_for_prevalence <- function(
         x, rank, relabel = FALSE, make.unique = TRUE, na.rm = FALSE,
         agg.na.rm = TRUE, ...){
-    # Check na.rm. It is not used in this function, it is only catched so that
+    # Check na.rm. It is not used in this function, it is only cached so that
```

Comment:
are you sure that "cached" is what is meant here - can you double check?

Reply:
Right, the word should be caught. Thanks